contrib/cirrus/runner.sh

```diff
@@ -233,8 +233,29 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH
+    then
+        msg "#####"
+        msg "Verifying no change adds new calls to io/ioutils."
+        local base diffs regex i_char
+        base=$(git merge-base $DEST_BRANCH $CIRRUS_CHANGE_IN_REPO)
+        diffs=$(git diff $base $CIRRUS_CHANGE_IN_REPO)
+        # Prevent this check from detecting itself (0x69 == "i")
+        regex=$(echo -e "^(\\+.+io/\x69outils)|(\\+.+\x69outils\\..+)")
+        if egrep -q "$regex"<<<"$diffs"; then
+            die "Found attempted use of deprecated ioutils:
+$(egrep -B5 -A5 "$regex"<<<"$diffs")"
```

Comment:
This is imperfect, but simple.  Hopefully 10-lines of context is enough for a PR author to find their problem.